detekt-cli/src/test/kotlin/io/gitlab/arturbosch/detekt/cli/runners/RunnerSpec.kt

```diff
@@ -299,4 +300,19 @@ class RunnerSpec {
             executeDetekt("--input", inputPath.toString(), "--max-issues", "2")
         }
     }
+
+    @Test
+    fun `regression - rule property, changed from comma separated string to list, throws on wrapped configs`() {
```

Comment:
```suggestion
    fun `does not fail when rule property type is changed from comma separated string to list`() {
```
I was a bit unclear if this test was added in order to document the existing behavior, or trying to add a test for the existing behavior.